easybuild/easyconfigs/g/gretl/gretl-2020a-foss-2019a.eb

```
@@ -0,0 +1,40 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Loris Bennett
+# ZEDAT, Freie Universitaet Berlin
+
+easyblock = 'ConfigureMake'
+
+name = 'gretl'
+version = '2020a'
+
+homepage = 'http://gretl.sourceforge.net'
+description = "A cross-platform software package for econometric analysis"
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'openmp': True}
```

gretl supports MPI, we can add `usempi` to compile it with mpi compilers
```suggestion
toolchainopts = {'openmp': True, 'usempi': True}
```